flexmeasures/data/schemas/account.py

```diff
@@ -29,9 +30,27 @@ class Meta:
 
     id = ma.auto_field(dump_only=True)
     name = ma.auto_field(required=True)
+    primary_color = ma.auto_field()
+    secondary_color = ma.auto_field()
+    logo_url = ma.auto_field()
```

Comment:
I think you can add a `required=False` to these fields instantiations, so that the validation will be skipped altogether if the fields are not present. And then you don't have to check for `None` values in the @validates class methods.

Reply:
"I used `allow_none=True` because I found that Marshmallow doesn't directly accept `required=False`. My IDE flagged an error, saying '`None is not callable`,' which prompted me to investigate. That's when I discovered this option, but even after adding it, the IDE continued showing the same warning. I'm not sure if this has any significant impact, as I don't have extensive experience with Flask. Have you encountered a case where this caused any problems in the past?

Reply:
I implemented my suggestion in https://github.com/FlexMeasures/flexmeasures/pull/1152/commits/489973d149632283a8981075c68f633ad0030d4a. I'm not sure what your IDE flagged and on what.